flow/flow.go

```diff
@@ -496,8 +518,18 @@ func (f *Flow) updateMetricsWithNetworkLayer(packet *gopacket.Packet) error {
 			f.Metric.BAPackets++
 			f.Metric.BABytes += int64(ipv6Packet.Length)
 		}
+
+		// update RTT
+		if f.state.network1stPacket == 0 {
+			f.state.network1stPacket = (*packet).Metadata().Timestamp.UnixNano()
+		} else {
+			if (f.RTT == 0) && (f.Network.A == ipv6Packet.DstIP.String()) {
+				f.RTT = (*packet).Metadata().Timestamp.UnixNano() - f.state.network1stPacket
```

Comment:
could be nice to have a specific value when the RTT calculation is not possible ex: sFlow. What about -1 ?

Reply:
RTT 0 is not possible (valid value)
For the issue with SFlow, I just add a patch that used the original packet timestamp